calico-cloud_versioned_docs/version-20-1/threat/container-threat-detection.mdx

```diff
@@ -85,11 +90,80 @@ EOF
 
 ### Monitor the Security Events page for malicious programs
 
-If a malicious or suspicious program is run within the cluster, it will be reported on the Security Events page of the
+If a malicious or suspicious program is run within the cluster, it will be reported on the **Security Events** page of the
 {{prodname}} UI.
 
 ![malware-alert-example](/img/calico-cloud/malware-alert-example.png)
 
+### Disable detectors
+
+All detectors are enabled by default. You may want to disable detectors that have a large performance impact and/or produce excessive false positive alerts.
```

Comment:
Up to this point, a user may have no idea of what a detector is, and we want to emphasis more the "showcasing capabilities" of the feature more than disabling individual detectors.

With this in mind, I would suggest we rephrase to something like:

```
Threat detection capabilities are provided by detectors that protect against specific [MITRE techniques](https://attack.mitre.org/techniques/enterprise/). 
Detectors are grouped by [MITRE tactics](https://attack.mitre.org/tactics/enterprise/) they protect against. Additional information for each detector can be found by clicking on a detector. 
All detectors are enabled by default, with the option of disabling specific detectors if needed.

<maybe INSERT SCREENSHOT> 
```

At this stage I don't think there is much to add about `Disable detector via UI`. It's fairly self-explanatory. I've lumped it in the first section. If we feel more detail is needed, I think we should balance it with info about "view detector details". 

I would only add one sub-section for applying a configuration to multiple clusters. Maybe something along the lines of:
```
#### Configure detectors via RuntimeSecurity Custom Resource

Detector settings configured through the UI are stored in the cluster's RuntimeSecurity Custom Resource. When configuring multiple clusters, the same configuration can be applied to all clusters.

Get the reference `detectorsConfig`:

    ```
    kubectl get runtimesecurities.operator.tigera.io default -o jsonpath="{.spec.detectorsConfig}" > detectorsConfig.json
    ```

Alternatively, can enable Container Threat Detection with the required configuration in one command:
    ```
    <kubectl apply command with settings>
    ```

```

Reply:
I initially had `#### Configure detectors via RuntimeSecurity Custom Resource` as `#### Configure detectors on multiple clusters`. Still not sure which one I prefer :sweat_smile:. The later is much more use case orientated, which might be a better fit for the docs.

Reply:
For the example, reading `detectorsConfig.json` from file could be a bit messy and distracting. We could just use and env var for simplicity and let the user figure out how to adapt it to their situation.

`detectorsConfig=$(kubectl get runtimesecurities.operator.tigera.io default -o jsonpath="{.spec.detectorsConfig}"\)`